runs/stream_logs.go

```diff
@@ -0,0 +1,63 @@
+package runs
+
+import (
+	"context"
+	"fmt"
+	"github.com/google/uuid"
+	"gopkg.in/nullstone-io/go-api-client.v0"
+	"gopkg.in/nullstone-io/go-api-client.v0/types"
+	"os"
+	"time"
+)
+
+// StreamLogs streams the logs from the server over a websocket
+// The logs are emitted to stdout
+func StreamLogs(ctx context.Context, cfg api.Config, workspace types.Workspace, newRun *types.Run) error {
+	// ctx already contains cancellation for Ctrl+C
+	// innerCtx will allow us to cancel when the run reaches a terminal status
+	innerCtx, cancelFn := context.WithCancel(ctx)
+	defer cancelFn()
+
+	fmt.Fprintln(os.Stdout, "Waiting for logs...")
+	client := api.Client{Config: cfg}
+	msgs, err := client.LiveLogs().Watch(innerCtx, workspace.StackId, newRun.Uid)
+	if err != nil {
+		return err
+	}
+	runCh := pollRun(innerCtx, cfg, workspace.StackId, newRun.Uid, time.Second)
```

Comment:
I'm not understanding the purpose of this. If we are receiving the logs via websockets, why do we need to poll to retrieve runs?

Reply:
I added a TODO to replace this with an EOF message pumped through the live logs.